### PR TITLE
Click farm icon my pools go to farm

### PR DESCRIPTION
**Pull request by hungdoansy**
# Description
In My Pools page, display the farm icons in the position card (both Classic & Elastic pools) when there is an active farm ongoing

### CODE REVIEW

File: src/pages/ProAmmPool/PositionListItem.tsx [hunk 1]
```diff
@@ -34,6 +34,25 @@ import { unwrappedToken } from 'utils/wrappedCurrency'
 
 import ContentLoader from './ContentLoader'
 
+const DisabledButtonOutlined = styled(ButtonOutlined)`
```
Comment (viet-nv) [inline src/pages/ProAmmPool/PositionListItem.tsx:37]: @namgold found the reason of disable button with mouse over tooltip. Just change props name from `disabled` to `isDisabled` or something else, it'll work

File: src/pages/ProAmmPool/PositionListItem.tsx [hunk 2]
```diff
@@ -270,8 +304,7 @@ function PositionListItem({ stakedLayout, farmAvailable, positionDetails, refe }
                 position={position}
                 tokenId={positionDetails.tokenId}
                 layout={1}
-                farmAvailable={farmAvailable}
-                disableCollectFee={!!stakedLiquidity}
```
Comment (viet-nv) [inline src/pages/ProAmmPool/PositionListItem.tsx:274]: `disableCollectFee` is not used anymore, isnt it?

**Review by viet-nv — CHANGES_REQUESTED** [review]

File: src/pages/ProAmmPool/PositionListItem.tsx (hunk 1 above)
Comment (namgold) [reply src/pages/ProAmmPool/PositionListItem.tsx:37]: or Diep's style: `$disabled`

Comment (viet-nv) [reply src/pages/ProAmmPool/PositionListItem.tsx:37]: I dont like $ sign on variable name. It likes JQuery...

Comment (namgold) [reply src/pages/ProAmmPool/PositionListItem.tsx:37]: it's like PHP :))

Comment (hungdoansy) [reply src/pages/ProAmmPool/PositionListItem.tsx:37]: Nice. Just found out that SC supports transient props. https://styled-components.com/docs/api#transient-props

File: src/pages/ProAmmPool/PositionListItem.tsx (hunk 2 above)
Comment (hungdoansy) [reply src/pages/ProAmmPool/PositionListItem.tsx:274]: Yeah, i removed it

File: src/pages/ProAmmPool/PositionListItem.tsx (hunk 1 above)
Comment (XiaoYhun) [reply src/pages/ProAmmPool/PositionListItem.tsx:37]: its a feature of styled-component not my style :))). It look unfamiliar at first point but later we will know it only use for SC styling

Comment (XiaoYhun) [reply src/pages/ProAmmPool/PositionListItem.tsx]: for example: if we use isDisabled and disabled at same time, how we know which is for styling which is prop of the component?

File: src/pages/ProAmmPool/PositionListItem.tsx [hunk 3]
```diff
@@ -294,37 +308,36 @@ function PositionListItem({ stakedLayout, farmAvailable, positionDetails, refe }
...
                   >
-                    <Text width="max-content" fontSize="14px" padding="8px">
-                      <Trans>Remove Liquidity</Trans>
-                    </Text>
-                  </ButtonOutlined>
+                    <ButtonOutlined
+                      style={{
+                        padding: '8px',
+                        width: '100%',
+                        pointerEvents: 'none',
```
Comment (viet-nv) [inline src/pages/ProAmmPool/PositionListItem.tsx:326]: should we update it directly on Button component? https://github.com/react-component/tooltip/issues/18#issuecomment-650864750

Comment (hungdoansy) [reply src/pages/ProAmmPool/PositionListItem.tsx:326]: I don't think we should as we can't interact with the buttons using pointerEvents none. Some buttons have (i) with tooltip and hovering (i) won't trigger tooltip as the event is skipped by the css value

**Review by XiaoYhun — DISMISSED** [review]

**Review by viet-nv — DISMISSED** [review]

**Review by viet-nv — APPROVED** [review]

File: src/pages/ProAmmPools/CardItem.tsx [hunk 4]
```diff
@@ -157,7 +157,15 @@ export default function ProAmmPoolCardItem({ pair, onShared, userPositions, idx
                   right: '16px',
                 }}
               >
-                <MouseoverTooltip text={t`Available for yield farming`}>
+                <MouseoverTooltip
+                  noArrow
+                  text={
+                    <Text>
+                      {t`Available for yield farming. Click`}{' '}
+                      <Link to={`/farms?tab=elastic&type=active&search=${pool.address}`}>here</Link> to go to the farm.
```
Comment (namgold) [inline src/pages/ProAmmPools/CardItem.tsx:165]: Partial translation.

Comment (namgold) [reply src/pages/ProAmmPools/CardItem.tsx:165]: Wrap all of this block into `<Trans></Trans>`

**Review by namgold — APPROVED** [review]

**Review by viet-nv — APPROVED** [review]

**Pull request merged**